src/goto-symex/goto_symex_state.cpp

```diff
@@ -776,16 +776,20 @@ ssa_exprt goto_symex_statet::add_object(
   const irep_idt l0_name = ssa.get_identifier();
   const std::size_t l1_index = index_generator(l0_name);
 
-  // save old L1 name, if any
```

Comment:
I'm not sure I appreciate that this comment has been removed.

Reply:
Put back in